spec/lib/nexmo_rails_spec.rb

```diff
@@ -25,5 +26,48 @@
       expect(client.private_key).to eq('NEXMO_PRIVATE_KEY')
     end
   end
+
+  describe 'private key initialization with file path defined' do
+
+    before do 
+      allow(File).to receive(:exists?).and_return(true)
+      described_class.setup do |config|
+        config.api_key           = 'NEXMO_API_KEY'
+        config.api_secret        = 'NEXMO_API_SECRET'
+        config.signature_secret  = 'NEXMO_API_SIGNATURE'
+        config.application_id    = 'NEXMO_APPLICATION_ID'
+        config.private_key       = './private.key'
+      end
+      allow(described_class.client.private_key).to receive(:present?).and_return(true)
```

Comment:
@benhayehudi I don't think this stub is necessary, it should pick up the value from the config block.